addons/mail/models/mail_channel_partner.py

```diff
@@ -45,3 +45,18 @@ def write(self, vals):
             if {'channel_id', 'partner_id', 'partner_email'} & set(vals):
                 raise AccessError(_('You can not write on this field'))
         return super(ChannelPartner, self).write(vals)
+
+    def mail_channel_partner_format(self, partner_infos=None):
+        members = self.read(['id', 'partner_id'])
```

Comment:
This read is useless (or counter-productive) because it won't use cache and it's overridden below with specific keys anyway, and wouldn't take advantage of what is already in `partner_infos`